css/theme-v4.css

```diff
@@ -1,13 +1,14 @@
 body {
-    --color-blue: #009cff;
+    --color-blue: #c3ccff;
```

Comment:
Any reasons we want to change `color-blue` and `color-blue-xd`? These are coming from the designer and are used in other blueprint for the web pages.

Reply:
Undoing.